### PR TITLE
Audit

### PR DESCRIPTION
**Pull request by cartercarlson**

### CODE REVIEW

File: src/test/OptionSettlement.t.sol [hunk 1]
```diff
@@ -43,18 +43,27 @@ contract OptionSettlementTest is DSTest, NFTreceiver {
     OptionSettlementEngine public engine;
 
     address public immutable ac = 0x36273803306a3C22bc848f8Db761e974697ece0d;
-    address public immutable wethAddress =
+    address public WETH =
```
Comment (0xAlcibiades) [inline src/test/OptionSettlement.t.sol:46]: Any reason these cant be immutable?

File: slither.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+```
```
Comment (0xAlcibiades) [inline slither.md:1]: No reason to include this here, can run slither on demand as needed, but also not useful to run on the libraries/false positive on the base64 encode

**Review by 0xAlcibiades — CHANGES_REQUESTED** [review]
Aside from removing the slither output, which we can run at any time, or even add to the CI pipeline (ideally) this looks good for an incremental, but hasn't really increased coverage.

File: slither.md (hunk 2 above)
Comment (0xAlcibiades) [reply slither.md:1]: See the commented out slither command in the CI pipeline, would be preferable to just get that working and have the output on each pipeline.

File: .solhintignore [hunk 3]
```diff
@@ -0,0 +1 @@
+test/OptionSettlement.t.sol
```
[the remark this answers is not in the document]
Comment (cartercarlson) [reply .solhintignore:1]: Fair enough - I will revise as so

**Pull request merged**